va/gsb.go

```diff
@@ -48,5 +48,5 @@ func (va *ValidationAuthorityImpl) IsSafeDomain(req *core.IsSafeDomainRequest) (
 	} else {
 		va.stats.Inc("VA.IsSafeDomain.Status.Bad", 1, 1.0)
 	}
-	return &core.IsSafeDomainResponse{status}, nil
+	return &core.IsSafeDomainResponse{IsSafe: status}, nil
```

Comment:
No, this looks correct.  `status` is true iff the safe browsing service returned the empty string, which indicates that the domain was not found on any of the lists (of bad domains).

It might be nice to rename `status` to something like `isSafe` just to be clear.